### PR TITLE
Add tables: FV0 and FDD (partially)

### PR DESCRIPTION
**Pull request by nburmaso**

### CODE REVIEW

File: Detectors/AOD/src/AODProducerWorkflowSpec.cxx [hunk 1]
```diff
@@ -807,16 +824,35 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
...
-  float dummyFV0AmplA[48] = {0.};
   uint8_t dummyTriggerMask = 0;
-  fv0aCursor(0,
-             dummyBC,
-             dummyFV0AmplA,
-             dummyTime,
-             dummyTriggerMask);
+
+  int nFV0ChannelsAside = o2::fv0::Geometry::getNumberOfReadoutChannels();
+  std::vector<float> vFV0Amplitudes(nFV0ChannelsAside, 0.);
```
Comment (shahor02) [inline Detectors/AOD/src/AODProducerWorkflowSpec.cxx:832]: `getNumberOfReadoutChannels()` is a constexpr, why not defining here 
`std::array<float, o2::fv0::Geometry::getNumberOfReadoutChannels()> FV0Amplitudes{};`

File: Detectors/AOD/src/AODProducerWorkflowSpec.cxx [hunk 2]
```diff
@@ -807,16 +824,35 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
...
+  for (auto& fv0RecPoint : fv0RecPoints) {
+    const auto channelData = fv0RecPoint.getBunchChannelData(fv0ChData);
+    for (auto& channel : channelData) {
+      vFV0Amplitudes[channel.channel] = channel.charge; // amplitude, mV
+    }
+    float aAmplitudesA[nFV0ChannelsAside];
+    for (int i = 0; i < nFV0ChannelsAside; i++) {
+      aAmplitudesA[i] = truncateFloatFraction(vFV0Amplitudes[i], mV0Amplitude);
```
Comment (shahor02) [inline Detectors/AOD/src/AODProducerWorkflowSpec.cxx:840]: and doing here directly 
```
FV0Amplitudes[channel.channel] = truncateFloatFraction(channel.charge, mV0Amplitude);
```

**Review by shahor02 — COMMENTED** [review]
hi @nburmaso 
looks fine, just a small comment below.

**Comment by sawenzel** [comment]
I am in favour of merging this. @nburmaso: The smaller corrections can be applied with the next iteration.

**Pull request merged**